{"uuid": "097e8a38-f8ec-40f3-8919-3aac7d4156e4", "children": ["9ddb31ed-35a1-4bec-bfe4-7faadf70650d", "8f4b078c-91a9-46f2-9d50-ba40bf3b578c", "331b028c-c2b7-4e38-9bc9-721ce7dac675", "bb51ee9c-67d8-4a8a-94ac-981428390571", "5176cba1-275c-4517-846b-0b738e73ff31", "e7f0e652-919a-47fc-87f5-b9d019f07efd", "09570144-521a-4516-a894-62546a8bcb1e", "37d3e913-8c7b-4652-b88b-4bb89198199e", "aff541e9-8438-44a9-93a3-84609ddfadf7", "7de167e4-0f20-402e-b783-9cb2e2a8b2c1", "03a5f5a3-7b4f-40b0-86bc-e85ada67c40f", "18841bd5-4f53-4da6-b68f-4b18e8e02156", "fd2b85cd-dd73-4f17-9673-b91393d849dc", "022a62e6-23fb-4568-a849-3dd14a550298", "845532d6-2c3a-461e-a172-516d00b5c0dd", "ec8f7f4f-998e-4ed9-915c-5d097c29d773", "4c6f85c0-1dd8-4830-8acb-15dd41c01f03", "0f21b136-a966-4e4e-8db8-418385b45fc9", "21420668-8e2f-4c86-9211-85543107112c", "1b2274de-f9c6-4972-b8bb-2bdd7baded31", "35e4f79b-123b-4eeb-a0c9-887d096e677d", "3bc8d5c9-0a0f-40b9-a210-32486662b3d5", "c1c98bcf-4d79-436f-8349-a0d0f2940b03", "dd2a7b40-a8cf-4e0b-bd15-0b116ead33fe", "a158e8d1-d145-4966-a2a4-15d67dd8ea6f", "0dd61d92-9d70-4b90-9cac-469a2cb9da05", "38a018cc-4e5e-4a31-8749-2d4adc49a488", "07512c03-0f35-483d-a970-cefc61ef5fd8", "10c2f633-2022-48ad-9342-1c4b0d51313d", "7b88395b-6631-4e94-855e-bef2265b6e83", "2e19f553-a899-4021-afde-0efee04d4b68", "c89f7105-c639-487f-a15c-b7b2d2f8ce64", "053b65d6-7713-448d-9bd9-e3e652eb46bc", "aa7b7639-915d-49b9-8efd-496397ae60d5", "e92a166c-582c-45f0-ab4b-e7fd1295713e", "ae739c73-5261-4a1e-a778-8d1614fe9e0b", "56d09d66-d2c3-4d61-a15f-9728a3cc7055", "c366681d-caf0-4db0-bbef-e4a4ee25c1bd", "81c067fd-ccc8-4cb7-ac73-2c6582f55788", "62d3bf8d-6833-45b7-8f17-26f11aa26a94", "3c302493-b46b-4acb-ab56-32308e932ad7", "f0edffe1-553c-47b0-96bc-fa7caa0fa682", "fdb9a903-f6f0-4d54-880e-6bbd2133dc14", "0e6974dd-0d52-46c2-86ee-c3eae2baa4b3"], "befores": [{"name": "session_login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: cannot find Chrome binary\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x009AD343+3134275]\n\tOrdinal0 [0x0089A131+2007345]\n\tOrdinal0 [0x0073AEC8+569032]\n\tOrdinal0 [0x006BC9B7+51639]\n\tOrdinal0 [0x006E0DFC+200188]\n\tOrdinal0 [0x006E0C1D+199709]\n\tOrdinal0 [0x006DEB0B+191243]\n\tOrdinal0 [0x006C2E77+77431]\n\tOrdinal0 [0x006C3E3E+81470]\n\tOrdinal0 [0x006C3DC9+81353]\n\tOrdinal0 [0x008B0C99+2100377]\n\tGetHandleVerifier [0x00B1B71A+1396954]\n\tGetHandleVerifier [0x00B1B399+1396057]\n\tGetHandleVerifier [0x00B270E6+1444518]\n\tGetHandleVerifier [0x00B1BCA8+1398376]\n\tOrdinal0 [0x008A7F11+2064145]\n\tOrdinal0 [0x008B22AB+2106027]\n\tOrdinal0 [0x008B23D1+2106321]\n\tOrdinal0 [0x008C4984+2181508]\n\tBaseThreadInitThunk [0x772B6359+25]\n\tRtlGetAppContainerNamedObjectPath [0x778A7C24+228]\n\tRtlGetAppContainerNamedObjectPath [0x778A7BF4+180]\n\n", "trace": "  File \"d:\\program files (x86)\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"d:\\program files (x86)\\lib\\site-packages\\_pytest\\fixtures.py\", line 1117, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"d:\\program files (x86)\\lib\\site-packages\\_pytest\\fixtures.py\", line 915, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\seleniumweb\\saas\\TestCase\\lab_test\\conftest.py\", line 17, in session_login\n    driver = webdriver.Chrome(chromedriver)\n  File \"d:\\program files (x86)\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 64, in __init__\n    super(WebDriver, self).__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n  File \"d:\\program files (x86)\\lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 79, in __init__\n    RemoteWebDriver.__init__(\n  File \"d:\\program files (x86)\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 183, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"d:\\program files (x86)\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 280, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"d:\\program files (x86)\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 349, in execute\n    self.error_handler.check_response(response)\n  File \"d:\\program files (x86)\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 240, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1601459283183, "stop": 1601459287092}], "start": 1601459283183, "stop": 1601459290473}